apps/projects/templates/a4_candy_projects/project_detail.html

```diff
@@ -1,324 +1,285 @@
 {% extends "base.html" %}
 {% load i18n rules react_follows thumbnail wagtailcore_tags a4_candy_project_tags offlineevent_tags contrib_tags ckeditor_tags static %}
 
-{% block title %}{{project.name}} &mdash; {{ block.super }}{% endblock %}
+{% block title %}{{ project.name }} — {{ block.super }}{% endblock title %}
 
 {% block social_meta_image %}
-{% if project.image %}
-<meta name="twitter:image" content="{{ project.image |thumbnail_url:'heroimage'}}">
-<meta name="linkedin:image" content="{{ project.image |thumbnail_url:'heroimage'}}">
-<meta property="og:image" content="{{ project.image |thumbnail_url:'heroimage'}}">
-<meta property="og:image:width" content="1500">
-<meta property="og:image:height" content="500">
-{% endif %}
-{% endblock %}
+    {% if project.image %}
+        <meta name="twitter:image" content="{{ project.image |thumbnail_url:'heroimage' }}">
+            <meta name="linkedin:image" content="{{ project.image |thumbnail_url:'heroimage' }}">
+                <meta property="og:image" content="{{ project.image |thumbnail_url:'heroimage' }}">
+                    <meta property="og:image:width" content="1500">
+                        <meta property="og:image:height" content="500">
+                        {% endif %}
```

Comment:
the indentation here seems off, should be on the same line as the `if`, same goes for the blocks below